cardano-node/cardano-node.cabal

```diff
@@ -126,9 +126,8 @@ executable cardano-node
   ghc-options:         -threaded
                        -Wall
                        -rtsopts
-                       "-with-rtsopts=-T"
+                       "-with-rtsopts=-T -I0"
                        -fno-warn-unticked-promoted-constructors
-                       -rtsopts
```

Comment:
Please keep `-rtsopts`

Reply:
That line is just removing a duplicate. It's already specified.